src/main/java/com/patternpedia/api/rest/controller/PatternController.java

```diff
@@ -239,6 +244,17 @@ EdgeResult getPatternLinksForPatternLanguageRoute(Pattern pattern, UUID patternL
         return links;
     }
 
+    @Operation(operationId = "getPatternByURI", responses = {@ApiResponse(responseCode = "200")}, description = "Retrieve patterns by pattern uri")
```

Comment:
this feature is implemented here as well: https://github.com/PatternAtlas/pattern-atlas-ui/pull/65

Reply:
it's  here: https://github.com/PatternAtlas/pattern-atlas-api/pull/16
Can you have a look at it and decide which version should be integrated into the master?

Reply:
I just talked to @pascalhuszar the changes regarding the `getPatternByURI` method should be fine